### PR TITLE
[WIP] Encryption of save file

### PR DESCRIPTION
**Pull request by 5jiji**
The pull request for the encryption of the save file is here!

It's still not ready but it will come ready at one point

*The file named crypt.py is all the function needed to encrypt decrypt and load the key, it will be merged with saveloader.py at one point*

### CODE REVIEW

File: crypt.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      keyFile = open('.key', 'r')
+      f = Fernet(keyFile.readline())
+      keyFile.close()
+    else:
+      CreateKey(False)
+
+def WinAttr(file):
+  import win32api
+  # Add the System and hidden attribute to the file
+  win32api.SetFileAttributes(file, 0x06)
```
[the remark this answers is not in the document]
Comment (luihum) [reply crypt.py:45]: also this will add a dependency, update the readme

[the remark this answers is not in the document]
Comment (luihum) [reply crypt.py:45]: I just tested on my Windows machine and for me it said module not found

[the remark this answers is not in the document]
Comment (luihum) [reply crypt.py:45]:
```
Traceback (most recent call last):
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\boot.py", line 146, in <module>
    boot()
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\boot.py", line 100, in boot
    systemexists = loadSystemSave(xobj.shortname)
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\saveloader.py", line 27, in loadSystemSave
    LoadKey()
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\crypt.py", line 33, in LoadKey
    CreateKey(True)
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\crypt.py", line 14, in CreateKey
    keyFile.write(key)
TypeError: write() argument must be str, not bytes
```

[the remark this answers is not in the document]
Comment (5jiji) [reply crypt.py:45]: I was sure there would be bugs

**Comment by luihum** [comment]
bruh 💀
```
Traceback (most recent call last):
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\boot.py", line 146, in <module>
    boot()
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\boot.py", line 100, in boot
    systemexists = loadSystemSave(xobj.shortname)
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\saveloader.py", line 29, in loadSystemSave
    csv_reader = csv.reader(Decrypt(f.readline()), delimiter=',')
  File "C:\Users\Luis\Desktop\Test\ProgressCLI95\crypt.py", line 8, in Decrypt
    return f.decrypt(data)
  File "C:\Users\Luis\AppData\Local\Programs\Python\Python310\lib\site-packages\cryptography\fernet.py", line 83, in decrypt
    timestamp, data = Fernet._get_unverified_token_data(token)
  File "C:\Users\Luis\AppData\Local\Programs\Python\Python310\lib\site-packages\cryptography\fernet.py", line 108, in _get_unverified_token_data
    utils._check_bytes("token", token)
  File "C:\Users\Luis\AppData\Local\Programs\Python\Python310\lib\site-packages\cryptography\utils.py", line 32, in _check_bytes
    raise TypeError("{} must be bytes".format(name))
TypeError: token must be bytes
```

**Pull request closed without merging**